src/main/resources/lang/strings_en.properties

```diff
@@ -202,7 +202,9 @@ config.bot_settings.dev_stuff.desc=Enabling this WILL make your bot use more cpu
 config.bot_settings.api=Browser/API, restart to apply
 config.bot_settings.api.desc=What browser or flash container to run the bot in
 config.bot_settings.fully_hide_api=Fully hide API
-config.bot_settings.fully_hide_api.desc=Will hide API in a different way. Lowers cpu usage. May not work properly for everyone
+config.bot_settings.save_main_gui_pos_and_size=Save main gui position and size
+config.bot_settings.save_main_gui_pos_and_size.desc=The bot will save its main gui's position and size so the next time when you start your bot,\n it will open up in the same place with the same size
+config.bot_settings.save_main_gui_screen_size=Save main gui screen size
```

Comment:
You seem to have deleted fully hide api description? Also you have main gui pos and size twice.

Try to make the name of the setting name itself shorter, "Save gui position" is enough, the user doesn't know what "main gui" or "config gui" means nor that you could save position and size independently, they'll just expect the feature to do what they want and save position and size

Also, i belive `\n` will not work if you don't do html and <br> in the tooltop for description, also, it feels a bit verbose:

"The bot will save its main gui's position and size so the next time when you start your bot, it will open up in the same place with the same size"
can be simplified to
"Save window location and size, and restore it next time you open the bot"